cfg/asd_demo.json

```diff
@@ -16,7 +16,11 @@
     "port": 10000,
     // optional asd_id is optional
     "asd_id" : "a globally unique id for the asd"
-    // optional, by default we use 99% of the file system capacity
+
+    // optional, limit's the disk usage of the asd to the specified capacity
+    // by default the asd uses the entire file system capacity
+    , "capacity" : 1000000000000
```

Comment:
if configuration would not have been automatic, getting the number of zeros right would be hard.